packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/types/FlutterAuthIUser.kt

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
The file is named `...IUser...`, but the class is `FlutterAuthUser`. Is the `I` intentional?

Reply:
Fixed.